corehq/apps/cleanup/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+
+class DeletedCouchDoc(models.Model):
+    doc_id = models.CharField(max_length=126)  # doc ids are unique across couch
```

Comment:
In addition to `doc_id`, would it also make sense to add `doc_type`? I think without that you would have to attempt the delete in every Couch database.

Reply:
It is is also not obviously true they are unique across all databases (although likely). They only need to be unique per couch database.

Reply:
Changed in 65aaed67e50c44c8df53e0cda8d16f04dc8944ee (and removed the comment Cal)